content/security/committers.md

```diff
@@ -1,110 +1,118 @@
+
+<h2 id="known">Known vulnerabilities<a class="headerlink" href="#known" title="Permanent link">&para;</a></h2>
 
 Projects with known, published vulnerabilities should provide information
 about those vulnerabilities on pages such as the
-[httpd security pages](http://httpd.apache.org/security_report.html). Provide a clear link on the project's home page to The
+[httpd security pages](http://httpd.apache.org/security_report.html). Provide a clear link on the project's home page to the
 security information.
 
 Do not enter details of security vulnerabilities in a project's public bug
 tracker unless the necessary configuration is in place to limit access to
 the issue to only the reporter and the project team.
 
-# Project specific security mailing lists
+<h2 id="lists">Project-specific security mailing lists<a class="headerlink" href="#lists" title="Permanent link">&para;</a></h2>
 
-Projects may also wish to create a project-specific security mailing list.
+Projects may wish to create a project-specific security mailing list.
 These take the name in the form `security@project.apache.org`, like
 `security@tomcat.apache.org`.
 
-When the infrastructure team creates these lists, they configure them to automatically copy
-all messages to `security@apache.org`. If is not,
-therefore, necessary to cc `security@apache.org` when sending mail to a project-specific security mailing list.
+When the infrastructure team creates project-specific security mailing lists, they configure them to copy
+all messages to `security@apache.org` automatically, so you do not have to
+cc `security@apache.org` when sending mail to such a list.
 
 We expect that a subset of project PMC members and committers will
 subscribe to the project-specific security mailing list. Do not use the list as a third-party notification system; non-committers should not
-be subscribed to the lists.
+be subscribed to the list.
 
-# Handling a possible vulnerability 
+<h2 id="possible">Handling a possible vulnerability<a class="headerlink" href="#possible" title="Permanent link">&para;</a></h2>
 
-A typical process for handling a new security vulnerability is as follows.
+Here is a typical process for handling a possible security vulnerability.
 Projects that wish to use other processes **may** do so, but **must** clearly and
-publicly document their process and have security@ review it before they begin using it.
+publicly document their process and have `security@` review it before they begin using it.
 
-<b><i>Note: this process changed in February 2021, when an internal portal
-for CVE allocation and submission became available</i></b>.
+**Note**: Do **not** make information about the vulnerability public until it isformally announced at the end of this process. That means, for example, that you should **not** create a Jira ticket to track the issue, since that would make the issue public.
+Messages associated with any commits should **not** make ANY reference to the
```

Comment:
I think the ANY can become any with the not being bolded.

Reply:
Updated in the patch